docs/tutorial.md

```diff
@@ -0,0 +1,48 @@
+Once you are in an environment with both cvmfs commands and pyhf supported, as well as the package installed, you are ready to run the translation
+
+### pyhf->Combine
+
+To go from a pyhf workspace to a datacard, you only need the workspace JSON file
+
+The translation can be run either via the CLI or with a python script. First, to run it in the CLI:
+
+```
+pyhf-to-combine $JSON_FILE_NAME
+```
+
+If you want to designate the names of the datacard and/or shapes files that are generated by the conversion, add the following flags respectively
+
+```
+--out-datacard $DATACARD_FILE_NAME
+--shape-file $SHAPES_FILE_NAME
+```
+
+To run the translation in a python script, run the following code:
+
+```
+from pyhf_combine_converter import pyhf_convert_to_datacard
+pyhf_convert_to_datacard.main($JSON_FILE)
+```
+
+### Combine->pyhf
+
+To run the translation in the other direction, the protocols are very similar. However, if the model has shapes, meaning there are samples with multiple bins and a shapes file is designated in the datacard, this shapes file must be present in the directory from which you are running the command. Otherwise there will be an error that the shapes file was not found.
+
+In the CLI:
+
+```
+combine-to-pyhf $DATACARD_FILE_NAME
+```
+
+To add the desired JSON file name, add the following flag:
+
+```
+--out-file $JSON_FILE_NAME
+```
+
+To run this conversion in a python script, run the following:
+
+```
+from pyhf_combine_converter import pyhf_converted_from_datacard
+pyhf_converted_from_datacard.main($DATACARD_FILE)
```

Comment:
It might be convenient to return the dict with the specification from this function, such that a user can directly go on and use that dict without having to load it from a file first.